src/Microsoft.Health.Checkpoints/ICheckpoint.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Checkpoints
+{
+    public interface ICheckpoint
+    {
+        string Identifier { get; set; }
+
+        DateTimeOffset LastProcessedDt { get; set; }
```

Comment:
nit: avoid the acronym in the property name

Reply:
updated